CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+* Friday 13th May 2022
+
+`Users needs are our strategy and our tactics` 
+
+## Features
+We added new endFormat (so your sound sounds perfect) this week we added Alexa preset 
+`code m = apiaudio.Mastering().create(scriptId=GLOBAL_SCRIPT_ID, endFormat="mp3_alexa")`
+
+## Enhancements
+* Improved pronunctuation of our custom voices. We had some pronunctuation issues reported by customers, this adds more natural sounding pauses to our voices.
```

Comment:
pronunciation?

Reply:
Yes - the work with adding gaps to speech in the voices (for our own custom voices).

Reply:
Ahh yes